src/pages/Booking/Seats/PeopleCount.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+function PeopleCount({
+  setPeopleTotalCount,
+  peopleIndex,
+  peopleTotalCount,
+  reset,
+  toggle,
+}) {
+  const [peopleCount, setPeopleCount] = useState(0);
+  const peoplePlus = () => {
+    if (peopleCount < 8) {
+      setPeopleCount(prev => prev + 1);
+    }
+  };
+
+  const peopleMinus = () => {
+    if (peopleCount > 0) {
+      setPeopleCount(prev => prev - 1);
+    }
+  };
+
+  useEffect(() => {
+    setPeopleTotalCount({ ...peopleTotalCount, [peopleIndex]: peopleCount });
+  }, [peopleCount]);
+
+  useEffect(() => {
+    setPeopleCount(0);
+  }, [toggle]);
```

Comment:
```suggestion
  const peoplePlus = () => {
    if (peopleTotalCount[peopleIndex] < 8) {
      setPeopleTotalCount(prev => ({
        ...prev,
        [peopleIndex]: peopleTotalCount[peopleIndex] + 1,
      }));
    }
  };

  const peopleMinus = () => {
    if (peopleTotalCount[peopleIndex] > 0) {
      setPeopleTotalCount(prev => ({
        ...prev,
        [peopleIndex]: peopleTotalCount[peopleIndex] - 1,
      }));
    }
  };
```

위에서 내려주는 state를 가지고 충분히 구현할 수 있는 부분입니다!
따로 state를 선언해서 사용하지 않아도 될 것 같네요!

위의 코드와 관련해서 변경되는 부분도 함께 수정해주세요!